c10/core/TensorImpl.h

```diff
@@ -65,7 +65,7 @@ inline std::vector<int64_t> ToVectorint64_t(const ArrayRef<int>& src) {
 /**
  * Return product of all dimensions starting from k
  */
-inline int64_t size_from_dim_(int k, IntArrayRef dims) {
```

Comment:
I think this can stay an int since we do not support int64 number of dims?

Reply:
The next function size_to_dim_ also uses int64 for k

Reply:
Reverted